PacketDefinitions420/PacketNotifier.cs

```diff
@@ -1263,10 +1263,7 @@ public void NotifyMissileReplication(IProjectile p)
             cast.IsClickCasted = false;
 
             cast.SpellSlot = p.OriginSpell != null ? p.OriginSpell.Slot : (byte)0x30;
-            cast.ManaCost = p.OriginSpell != null ? p.OriginSpell.SpellData.ManaCost[p.OriginSpell.Level] : 0f;
             cast.SpellCastLaunchPosition = p.GetPosition3D();
-            cast.AmmoUsed = p.OriginSpell != null ? p.OriginSpell.SpellData.AmmoUsed[p.OriginSpell.Level] : 0;
```

Comment:
I didn't really understand, why these two lines get removed? Is it related to just the 'display' (which you can change with the tooltips packet)?

Reply:
They don't need to be there because it is for a projectile packet.